charts/fluentd-elasticsearch/values.yaml

```diff
@@ -150,6 +150,11 @@ elasticsearch:
 # by example you can add -vv to launch with trace log
 fluentdArgs: "--no-supervisor -q"
 
+# Specifies the output format of fluentd's own log messages. Set to
+# "json" to enable structured logging.
+# See https://docs.fluentd.org/deployment/logging#log-format
+# fluentdLogFormat: "json"
```

Comment:
i would prefer to enable this setting and set it to the default "text".

Reply:
If we set a default here, then it will "shadow" fluentd's own default setting. I thought it would be best for the Helm chart to just take the fluentd default unless someone explicitly tells us what they want.

This way, if fluentd changes their default to "json", the chart will just pick up that behavior change... which feels natural to me.

Reply:
@monotek Oops. Should have pinged you in my comment. I would suggest that we "default to the default" as explained above.

Reply:
the other options are already using the default most of the time so i'd like to have this consistent.

Reply:
Fair enough. I've pushed a new commit, which sets the default to `text` to match fluentd's default, and updated the README accordingly.